awswrangler/_databases.py

```diff
@@ -71,6 +88,7 @@ def _get_connection_attributes_from_secrets_manager(
         host=secret_value["host"],
         port=secret_value["port"],
         database=_dbname,
+        ssl_context=None,
```

Comment:
I guess there is really no way to specify/pass the ca data via secrets manager... Do you think we should clarify the docs in that regard?

Reply:
Doesn't hurt to clarify, of course.